e2e/browser/test/fixtures.ts

```diff
@@ -241,6 +241,91 @@ const createClientResource = async (
 const clientApplicationUrl =
   process.env.E2E_DEMO_CLIENT_APP_URL ?? "http://localhost:3001/";
 
+export interface ISeedPodResponse {
```

Comment:
Not sure if I understand the naming of this type. What is an `ISeedPodResponse` abstracting, just the session used to access the pod we  are using in the test?

Reply:
The `seedPod` function generates a client ID document and a resource which only that application can access. The URIs of those to resources are the main things used; the session that is returned is used to interact with those resources in some cases.